crates/router/src/core/user/dashboard_metadata.rs

```
@@ -519,6 +519,42 @@ async fn insert_metadata(
                 }
             }
 
+            // Hubspot integration
+            let hubspotbody = state
+                .crm_client
+                .make_body(CRMPayload {
+                    legal_business_name: data.legal_business_name,
+                    business_label: data.business_label,
+                    business_location: data.business_location,
+                    display_name: data.display_name,
+                    poc_email: data.poc_email,
+                    business_type: data.business_type,
+                    business_identifier: data.business_identifier,
+                    business_website: data.business_website,
+                    poc_name: data.poc_name,
+                    poc_contact: data.poc_contact,
+                    comments: data.comments,
+                    is_completed: data.is_completed,
+                    business_country_name: data.business_country_name,
+                })
+                .await;
+            let base_url = user_utils::get_base_url(state);
+            let hubspot_request = state
+                .crm_client
+                .make_request(hubspotbody, base_url.to_string())
+                .await;
+
+            let _ = state
+                .crm_client
+                .send_request(&state.conf.proxy, hubspot_request)
+                .await
+                .inspect_err(|_| {
```

Don't discard error, and do error/warn logging instead of info

---

> Don't discard error
We do not want the prodIntent flow to stop if the crm fails hence discarding the error